paasta_tools/cli/cmds/spark_run.py

```diff
@@ -1108,6 +1118,13 @@ def paasta_spark_run(args):
         needs_docker_cfg=needs_docker_cfg,
         auto_set_temporary_credentials_provider=auto_set_temporary_credentials_provider,
     )
+    # Experimental: TODO: Move to service_configuration_lib once confirmed that there are no issues
+    # Enable AQE: Adaptive Query Execution
+    if "spark.sql.adaptive.enabled" not in spark_conf:
+        spark_conf["spark.sql.adaptive.enabled"] = "true"
```

Comment:
ultimately I think we want this to go in spark-on-paasta right?  I want to avoid a situation where we have configs being specified from all these different places since that will make it hard to track things down later.

Reply:
I think ultimately, I would prefer to have all configs being either set here (paasta/spark_run.py) or service_configuration_lib (where we set other dynamically calculated spark configs). 
spark-on-paasta poses the limitation to rollout configs to all batches easily.

As on now keeping it here, as it's easier to rollout and rollback. Once the experiment ok for a week, I will move this to `service_configuration_lib` where some of the other spark configs are set. 

Agreed on the part, that we want to avoid spark configs being spread out into different repositories.